packages/forge2d/lib/src/dynamics/contacts/contact.dart

```diff
@@ -162,18 +162,21 @@ abstract class Contact {
             this.indexB == indexA);
   }
 
-  /// Enable/disable this contact. This can be used inside the pre-solve contact listener. The
-  /// contact is only disabled for the current time step (or sub-step in continuous collisions).
-  void setEnabled(bool enable) {
+  /// Enable or disable this contact.
+  ///
+  /// This can be used inside [ContactListener.preSolve]. The contact is
+  /// only disabled for the current time step (or sub-step in continuous
+  /// collisions).
+  set enabled(bool enable) {
```

Comment:
This one is pretty clear, compared to the other one, but both since it was named `isEnabled` before and to make the user dead sure that it is a boolean I'd name it `isEnabled`.